# Results of the work 'Identification and characterization of the *Cicer arietinum* aldehyde dehydrogenase superfamily'


## Identification and characterization of aldehyde dehydrogenases of C. arietinum

The identification process of the CaALDH is shown in Figure 1. Considering the species of the search, the model plant *Arabidopsis thaliana* and the legumes *Glycine max* and *Medicago truncatula*, we can conclude to have covered an adequate spectrum of species that, by homology of sequences, will allow us to identify all the aldehyde dehydrogenases of *Cicer arietinum*.

The availability of the complete chickpea genome together with database searches allowed the identification of 37 chickpea ALDH proteins encoded by 29 *CaALDH* genes (Table 1). These genes encode proteins with a range of 134 (CaALDH3H1j) to 755 (CaALDH18B3d) aa of length. The number of exons of the *CaALDH* genes varies from 3 (CaALDH3H1j) to 21 (CaALDH18B3a, CaALDH18B3d and CaALDH18B3e). The corresponding molecular weights range between 15.07 and 81.90 kDa; and the predicted isoelectric points between 4.34 and 9.49 (Table 2). The wide range of pI suggests that chickpea ALDH proteins work in very different subcellular environments.

We observed that families 5, 6, 11, 12 and 22 are defined by a single gene in chickpea, similar to *Arabidopsis, O. sativa, S. italica, S. bicolor, E. parvulum* and *E. salsugineum* (Table 3); suggesting that these families constitute ALDH house-keeping genes, involved in the plants central metabolism and the preservation of non-toxic aldehydes levels. Families 2, 3 and 18 are comparatively abundant in *C. arietinum*. Family 3 (10 genes) and family 18 (6 genes) have a higher number of members in chickpea than any plant species described so far. ALDH3 proteins constitute one of the most widespread and diverse groups of ALDH genes in plant species (Brocker et al., 2013).
There are no *CaALDH* genes in families 19, 21, 23 and 24. Families 21 and 23 have been found to contain only primitive terrestrial plant genes (Chen et al., 2002), and ALDH24 appears only in the single-celled alga *C. reinhardtii* (Wood & Duff, 2009); suggesting that these three families may have played an important role in the evolution of lower plants and subsequently were lost in higher plants. Family 19 is found only in tomato (*S. lycopersicum*), so it evolved specifically in this lineage (Jimenez-Lopez et al., 2016).

![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/Esquema_CaALDH1.jpg)
**Figure 2**. CaALDH identification scheme. Own functions appear in color. Proteins from *Arabidopsis, G. max* and *Medicago* were used as a query in BLASTP searches against of the chickpea reference genome. The result of this was cleaned with R scripts to avoid repetitions. Chickpea proteins without the ALDH domains were deleted and another BLASTP was run against the *Cicer* genome to ensure the inclusion of every CaALDH and the detection of the unpredicted ones. This resulted in 37 aldehyde dehydrogenases encoded by the chickpea genome.

**Tabla 1**. Chickpea ALDH proteins classification.
![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/TABLA1.jpg)


*ALDH18* genes encode Δ1 -pyrroline-5-carboxylate synthetase (P5CS), defined as ALDH-like proteins (Sophos & Vasiliou, 2003). They are involved in proline biosynthesis (Igarashi et al., 1997), whose accumulation has adaptive roles in tolerance to biotic/abiotic stresses (Verbruggen & Hermans, 2008). Li et al. (2013) conclude that family 18 is the group that differs the most between species. The gene structure of the members of this family is different from that of the other families and presents additional domains AA-kinase; lacking the preserved ALDH active sites as shown by the results (Table 1). In addition, it is believed that the balance between proline biosynthesis and degradation is essential for its osmoprotective functions. Δ1 -pyrroline-5-carboxylate dehydrogenase (P5CDH) acts on the degradation, which is an ALDH12 protein. This degradation occurs in the mitochondria (Verbruggen & Hermans, 2008), which agrees with the results obtained in the prediction of subcellular localization (Table 1).

**Table 2**. CaALDH Characterization.
![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/Tabla%202.jpg)

**Table 3**. Members of ALDH families identified in plants, humans and fungi.
![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/All%20species%20ALDH%20families.jpg)



Most genes in family 3 seem to be regulated by the abscisic acid (ABA) stress response pathway, they get expressed in response to abiotic stresses (Brocker et al., 2013, Kirch et al., 2004). Family 3 in plants have diverged to six subfamilies: 3E, 3F, 3H, 3I, 3J and 3K. Two of these subfamilies are found in chickpea (3F and 3H). The 3H1 subfamily is very expanded with 10 members encoded by 7 genes, while 3F1 has 3 members encoded by 3 genes. Missihoun et al. (2012) have postulated the abundance of ALDH3s proteins as result of a complex expression pattern of their genes regulated by gene-splicing or alternative promoters. Based on the differences in expression and response in *A. thaliana* (Kirch et al., 2004), it was suggested that the isoforms of ALDH3 have evolved as a consequence of functional specialization in specific tissues and subcellular organelles; which fits with the diversity of subcellular localization predicted for the chickpea proteins of this family (Table 1). The algae described so far, *C. reinhardtii* unicellular and *V. carteri*  colonial, lack family 3; suggesting that the expansion and diversification of this gene family occurred along with the evolutionary movement of aquatic plants (Brocker et al., 2013).

ScanProsite analysis showed that the characteristic domains PS00687 and PF00070 were not found on the 37 ALDH sequences: 12 of the 37 contain both domains; 5 of them contain only domain PS00687 and 2 of them contain only PS00070. Some of the proteins that do not contain these domains are found in subfamilies 3F1, 3H1, and family 18. Therefore, it was necessary to make other alternative searches to identify these ALDH. In family 18, the domains PS00902 (Glutamate 5 kinase signature) and PS01223 (γ-glutamyl phosphate reductase signature) appear.

All ALDH gene families identified in higher plants such as *Arabidopsis* are present in *C. arietinum*, with the exception of ALDH19 present only in *S. lycopersicum* (Table 3). The chickpea, with 29 CaALDH genes, is the third one along with tomato with the highest number of ALDH genes. Above tham is apple tree with 39 genes and cotton with 30 genes. *C. arietinum* seems to have additional ALDH proteins for stress response. Families 3 and 18 are particularly abundant, which may be significant for carrying out the detoxification of aldehyde molecules generated under different stresses and maintaining the homeostasis of reducing equivalents.

*CaALDH* genes were mapped to the chickpea genome in order to understand the chromosomal distribution. Based on the available genome assembly of *C. arietinum*, 23 of the 29 genes are distributed in seven of the eight chromosomes. We find genes that encode ALDH from different families on the same chromosome. We could not map genes whose proteins have short sequence length (<272 aa). Chromosomes 6 and 7 contain 52% of the mapped genes. No *CaALDH* gene is mapped on chromosome 2 (Figure 3). The six *CaALDH* genes that can not be mapped in the reference genome (Table 2) could be mitochondrial genes; since the mitochondria is not found in the distribution of the reference genome in the NCBI.

![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/Todos%20chr.jpg)
**Figure 3**. Genomic distribution of *CaALDH* genes on the chickpea chromosomes. Chromosome number and sizes (Mb) are indicated above and below each bar, respectively. Only chromosomes with *CaALDH* genes are represented.



## Classification of *C. arietinum* aldehyde dehydrogenases 

The final classification is found in Table 1. The classification system established by the AGNC follows the guidelines of the Human Gene Nomenclature (http://www.genenames.org/guidelines.html) that recommends its use for every specie. All this information can be found in the Aldehyde Dehydrogenase Superfamily Database (http://www.aldh.org). Since all the information is referenced and governed by the animal kingdom, the ALDH classification in plant species is subordinated to what was previously described. In turn, a protein will belong to the family and subfamily, even have the same gene number, of the ALDH already identified with which it has more homology; to facilitate next comparative studies. As an example we have the case of the CaALDH6B2 protein, which tests the non-sequential allocation of the gene number. Thus, the definition of the AGNC does not seem to meet the usual practices.

In conclusion, of the 24 existing ALDH families, the CaALDH are included in 10 of the 14 ALDH families previously described in plant species (2, 3, 5, 6, 7, 10, 12, 18 and 22) (Table 3). To achieve a safe classification of less clarified families, such as 18, it was necessary to make alignments between them and look at the set of properties that correspond to their function and subcellular location (Table 1).


## Phylogeny

In order to study the phylogenetic relationships between the chickpea and *Medicago* ALDH, we carry out a multiple alignment between their sequences. Both species are closely related; *C. arietinum* diverged from *M. truncatula* ~ 10-20 million years ago (Varshney et al., 2013). This phylogenetic analysis gives us a view of the changes occurred during the speciation.

The phylogenetic tree shows 2 sister pairs between chickpea proteins, suggesting recent duplications after the two species separated. On the other hand we found 14 sister pairs CaALDH - MtALDH indicating that in these sequences there are no recent duplications. There is a single sister pair between *Medicago* proteins that suggests a deletion in the chickpea genome.
The ALDH18 sequences appear in a separated cluster from the rest of the sequences. The sequences of the three problematic proteins in the classification (18B3h, 18B3f and 18B3g, of short aa length) appear in a subgroup next to a MtALDH18 and CaALDH18B3d. The rest of CaALDH18 appear with their respective *Medicago* sister pairs. Thus, we hypothesize that these three *C. arietinum* ALDH18 could be the result of duplications subsequent to speciation.
Below is the cluster that corresponds to the ALDH12 sequences. Considering the function of families 12 and 18 in the same metabolic pathway, it makes sense that their sequence also differs from the rest of sequences of the analysis.
The 100% identity of the *Cicer-Medicago* sister pair in the ALDH7 sequences fits with the idea that this family is one of the most conserved eukaryotic proteins in evolution (Missihoun et al., 2014).

The rest of sequences are included in a subdivided cluster. In the ALDH6 sequences subgroup we find the opposite case of the family 18: *Medicago* contains two MtALDH6B2, and *Cicer* one CaALDH6B2. The evolutionary pressure could cause the loss of one of them after speciation. We have the same case in the subgroups of the sequences ALDH11, ALDH10 (or BADH) and ALDH2B4, but in this last there are two isoforms in *Cicer*, an event that does not occur in *Medicago*.
The ALDH3 sequences are contained in another subgroup. The *Cicer-Cicer* and *Medicago-Medicago* sister pairs in the 3H1 subfamily indicate that a series of duplications and deletions occurred after speciation.
In the ALDH22 sequences cluster we found the uncharacterized aldehyde dehydrogenase protein of *Medicago* associated with the *Cicer-Medicago* sister pair; this protein could belong to the family 22.


![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/phylogenetic%20tree%20MtALDH-%20CaALDH.png)
**Figure 4**. Analysis of the phylogenetic relationships of ALDH in *C. arietinum* and *M. truncatula*. 29 CaALDH proteins and 28 MtALDH proteins. The percentage of replicated trees in which the proteins were associated in the same cluster in the bootstrap test (1000 replicates) is indicated next to the branches. The CaALDH are marked with a diamond, the colors indicate each family and subfamily.


## Duplication analysis

To corroborate the hypotheses of duplications indicated throughout the work, an analysis was performed with the Circoletto tool.
The sequences of the proteins of the 3F1 subfamily indicate a high identity (Figure 5). In the phylogenetic tree, Figure 3, each one appears with its respective sister pair of *Medicago*. Considering both results, if there had been an event of duplication in these genes, this would have occurred before the speciation process.
In phylogeny, the sequences of the proteins MtALDH3H1.11, CaALDH3H1c and CaALDH3H1d are grouped in the same cluster (Figure 3). In Figure 5 these two chickpea proteins show an identity > 99%. The results of both analysis coincide and support the hypothesis of the event of duplication after speciation.
Short aa sequences are not analyzed with Circoletto, so this does not help to elucidate the problems of families 3 and 18 proteins.

![](https://github.com/RocioCarmonaMolero/TFMweb/blob/master/duplication%20analysis_1genperlocus.png)
**Figure 5**. Analysis of duplications between CaALDH with the Circoletto tool; 1 gene per locus. Colors: green identities ≤ 95%, orange ≤ 99% and red> 99%.
